LeapEngine/SceneGraph/GameObject.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
 #include "../Components/Component.h"
+#include "Debug.h"
+#include "../Utils/ReflectionUtils.h"
```

Comment:
Remove the ../Utils/
This path is now getting hardcoded right to the other directory. The Utils library is included for this engine.
So use #include "ReflectionUtils.h"